adafruit_datetime.py

```diff
@@ -1520,18 +1520,10 @@ def toordinal(self) -> int:
         return _ymd2ord(self._year, self._month, self._day)
 
     def timestamp(self) -> float:
-        """Return POSIX timestamp as float.
-
-        Note that Floats on most boards are encoded in 30 bits
-        internally, with effectively 22 bits of precision. As a result,
-        for modern dates this value can be off by several minutes.
-        As a workaround you can access the function ``_mktime()``
-        to get an int version of the timestamp.
-        """
+        """Return POSIX timestamp as float."""
```

Comment:
You reverted the doscstring to saying "as float".
```suggestion
        """Return POSIX timestamp as int, similar to the value returned by ``time.time()``."""
```

I like the way the C python version mentions time.time (which is an int too on CP).
> Return POSIX timestamp corresponding to the [datetime](https://docs.python.org/3/library/datetime.html#datetime.datetime) instance. The return value is a [float](https://docs.python.org/3/library/functions.html#float) similar to that returned by [time.time()](https://docs.python.org/3/library/time.html#time.time).